backend/tests/admin/test_api.py

```diff
@@ -69,8 +69,9 @@ def test_get_users(self):
         self.login_admin()
         resp = self.app.get("/api/v1/admin/users")
         self.assert200(resp)
+        print(resp.json)
```

Comment:
No biggie since it's a test, but got a print here.

Reply:
These can be nice since they only render if the test fails